garage/tf/baselines/__init__.py

```diff
@@ -7,6 +9,7 @@
     GaussianMLPBaselineWithModel)
 
 __all__ = [
+    'ContinuousMLPBaselineWithModel',
     'DeterministicMLPBaseline',
```

Comment:
is the new name for this one `ContinuousMLPBaseline` or `DiscreteMLPBaseline`?

Reply:
This is the new name for the original `DiscreteMLPBaseline` because there was no `ContinuousMLPBaseline` in garage before.